src/__tests__/Storyshots.test.tsx

```diff
@@ -0,0 +1,10 @@
+import initStoryshots from "@storybook/addon-storyshots"
+
+initStoryshots({
+  storyNameRegex: /Gene Follow/,
+  test: ({ story, context }) => {
+    // console.log(story)
+    // console.log(context)
+    // const storyElement = story.render(context)
```

Comment:
@kanaabe Is passing in `context` as the argument to `story.render(…)` the documented/required thing to do? If so, can you link me to it?

Reply:
Nope, `context` is not necessary! It was using in the docs here so I had it but it works without: https://github.com/storybooks/storybook/tree/master/addons/storyshots#example